### DISCUSSION TITLE
How long will it take for EasyMotion?

### DISCUSSION THREAD

**Discussion by txtyash** [helix-editor/helix#3667] [Polls]
I've been waiting on EasyMotion since very long. I love that feature. I'm wondering if there's any work being done...

**Comment by Omnikar** [comment]
I haven't posted any updates about it, but I have begun redoing #1162 in an improved way and plan to eventually open a replacement PR. Depending on how busy I end up being, I'd say 1–3 months is a reasonable estimate for how long it should take.

**Reply by Omnikar** [reply]
#3791 is now open.

**Reply by James-Firth** [reply]
~For anyone else searching for EasyMotion #3791 and been replaced by #5340 and is open at the time of posting~
Look at #8875 that is mentioned by kirawi below

**Reply by kirawi** [reply]
That PR has been superseded by https://github.com/helix-editor/helix/pull/8875

**Reply by James-Firth** [reply]
Whoops missed that update, thanks!

**Reply by James-Firth** [reply]
Since I'd been tracking this closely FYI future readers: That PR has been merged into master now via commit https://github.com/helix-editor/helix/commit/b46064b8c469c85b9626dba60728e23798354ed8

**Comment by adrian5** [comment]
I too am waiting for this feature, as it has become a deeply ingrained habit to navigate this way. There a few things keeping me with nvim, and this is among the biggest.
